libkbfs/md_util.go

```diff
@@ -158,6 +158,21 @@ func getMDRange(ctx context.Context, config Config, id TlfID, bid BranchID,
 	return rmds, nil
 }
 
+// getSingleMD returns an MD that is required to exist.
+func getSingleMD(ctx context.Context, config Config, id TlfID, bid BranchID,
```

Comment:
Yep, this PR already includes all the ones I could find.  (Only 3 total.)  Did I miss one?